apps/plotly_examples/python-scripts/generate_plotly_schema.py

```diff
@@ -3,6 +3,7 @@
 from pydantic import BaseModel
 from azure.identity import ManagedIdentityCredential, DefaultAzureCredential, VisualStudioCodeCredential, InteractiveBrowserCredential, get_bearer_token_provider
 from openai import AzureOpenAI
+import random
```

Comment:
Add a text file mentioning which schema id correspond to which locale, large data etc.